ext/XS-APItest/APItest.xs

```diff
@@ -6795,6 +6795,31 @@ test_toTITLE_utf8(SV * p, int type)
     OUTPUT:
         RETVAL
 
+AV *
+test_delimcpy(SV * from_sv, STRLEN trunc_from, char delim, STRLEN to_len, STRLEN trunc_to, char poison = '?')
```

Comment:
It's not a big deal for test code like this (unless we valgrind or ASAN), but the default-for-backward-compatability AV* typemap leaks.

This should include 
```
AV*	T_AVREF_REFCOUNT_FIXED
```

in the typemap to get the non-leaky version.

Reply:
grepping, I didn't find an example of how to do this.  There are a bunch of functions in this .xs that return AV's

Reply:
The fix only needs the line above to be added to the top section of `ext/XS-APItest/typemap`.

Unless they have an extra sv_2mortal() or used the fixed typemap entry they'll leak the AV, see T_AVREF in perlxstypemap.